Library/PTPusher.h

```diff
@@ -81,11 +82,22 @@ extern NSString *const PTPusherErrorUnderlyingEventKey;
 ///------------------------------------------------------------------------------------/
 
 /** The object that acts as the delegate for the receiving instance.
- 
+
  The delegate must implement the PTPusherDelegate protocol. The delegate is not retained.
  */
 @property (nonatomic, weak) id<PTPusherDelegate> delegate;
 
+/** This property is deprecated and will be ignored.
+ *
+ * The client will always attempt to reconnect when it disconnects as long as the error
+ * code it disconnected with permits automatic reconnection (see the Pusher protocol documentation
+ * for an overview of error codes: http://pusher.com/docs/pusher_protocol).
+ *
+ * If you want to prevent the client from automatically reconnecting, you can do so by returning
+ * NO from the delegate method pusher:connectionWillConnect:.
+ */
+@property (nonatomic, assign, getter=shouldReconnectAutomatically) BOOL reconnectAutomatically __PUSHER_DEPRECATED__;
+
```

Comment:
Hmmm - think that might have slipped in accidentally as part of the rebase. I'll have a look